src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/DataElementsCard.tsx

```diff
@@ -25,7 +28,85 @@ const DataElementsCard = (props: {
   const negationRationale =
     selectedDataElement?.hasOwnProperty("negationRationale");
   // https://ecqi.healthit.gov/mcw/2020/qdm-attribute/negationrationale.html  (list of all categories that use negation rationale)
-
+  // from here we know the type, we need to go through the dataElements to matchTypes
+  // attributes section
+  const [displayAttributes, setDisplayAttributes] = useState([]);
+  // codes section
+  const [codesChips, setCodesChips] = useState([]);
+  const [localSelectedDataElement, setLocalSelectedDataElement] =
+    useState(null);
+  useEffect(() => {
+    if (selectedDataElement) {
+      const displayAttributes = [];
+      const codesChips = [];
+      const qdmType = selectedDataElement?._type; // match against for attributes
+      const model = qdmType.split("QDM::")[1];
+      const getModel = cqmModels[model];
+      const modeledEl = new getModel(selectedDataElement);
+      setLocalSelectedDataElement(modeledEl);
+      modeledEl.schema.eachPath((path, info) => {
+        if (!SKIP_ATTRIBUTES.includes(path) && modeledEl[path]) {
+          if (info.instance === "Array") {
+            modeledEl[path].forEach((elem, index) => {
+              if (path == "relatedTo") {
+                let id = elem;
+                const display = getDisplayFromId(modeledEl, id);
+                let value = `${stringifyValue(
+                  display?.description,
+                  true
+                )} ${stringifyValue(display?.timing, true)}}`;
+                displayAttributes.push({
+                  name: path,
+                  title: _.startCase(path),
+                  value: value,
+                  isArrayValue: true,
+                  index: index,
+                });
+              } else {
+                displayAttributes.push({
+                  name: path,
+                  title: _.startCase(path),
+                  // this is wrong.
```

Comment:
I have to ask, why is this a comment?!

Reply:
Oh, this case is for the complex attributes. I do not have it working right at this time and broke it into another ticket